### PR TITLE
Issue 3332 final

### PR DESCRIPTION
**Pull request by ps-sp**
#3332. Added `rb` `rt` `rtc` and `rp` HTML tags.

### CODE REVIEW

**Comment by ps-sp** [comment]
@romani @rnveach please have a look. (unrelated UTs were unsuccessful on my local which didn't make much sense, let's see what the CIs say). Thanks.

**Comment by romani** [comment]
there is problem with dtd files load from web, fix is ready, am I waiting PRs to confirm fix.

**Comment by romani** [comment]
@ps-sp ,

please address this (appveyor):
```
[INFO] --- maven-pmd-plugin:3.9.0:check (default) @ checkstyle ---
[INFO] PMD Failure: com.puppycrawl.tools.checkstyle.grammars.javadoc.GeneratedJavadocTokenTypesTest:46 
Rule:ExcessiveMethodLength Priority:3 Avoid really long methods..
[INFO] PMD Failure: com.puppycrawl.tools.checkstyle.grammars.javadoc.GeneratedJavadocTokenTypesTest:46 
Rule:NcssCount Priority:3 The method 'testTokenNumbers()' has a NCSS line count of 102..
```

**Comment by romani** [comment]
fix for Import UT is merged, please rebase on latest master, no UT failures are expected.

**Comment by romani** [comment]
@ps-sp , please just rebase your commit, you should have no issues.
It will pass CI so we will be safe to merge your changes.

-------------

>
```
 -        assertEquals(msg, 96, JavadocTokenTypes.HTML_TAG_NAME);
...
+        assertEquals(msg, 100, JavadocTokenTypes.HTML_TAG_NAME);
```

each tag addition will result in damaging of JavadocTokenTypes, and HTML_TAG_NAME will be  used a lot in Checks as it is very general tag.
kind of sad , ... but we will think about it a bit later.

File: src/test/java/com/puppycrawl/tools/checkstyle/utils/JavadocUtilTest.java [hunk 1]
```diff
@@ -357,9 +357,9 @@ public void testGetPreviousSibling() {
     }
 
     @Test
-    public void testGetTokenNames() {
+    public void testGetLastTokenName() {
         assertEquals("Unexpected token name",
-            "OPTGROUP", JavadocUtil.getTokenName(10082));
+            "RP", JavadocUtil.getTokenName(10094));
```
Comment (ps-sp) [inline src/test/java/com/puppycrawl/tools/checkstyle/utils/JavadocUtilTest.java]: This test was redundant. Was previously equivalent to `JavadocUtilsTest#testGetTokenNameForId`. Hence modified accordingly.

**Pull request merged**